gapic/packaging/dependencies.yaml

```diff
@@ -91,6 +91,9 @@ google-iam-v1_version:
   ruby:
     name_override: grpc-google-iam
     lower: '0.6.8'
+  java:
+    name_override: grpc-google-iam-v1
```

Comment:
But for Java, we just always prepend "grpc-" to every package, which seems like we could do in the transformer without having to do any name overrides in the config.